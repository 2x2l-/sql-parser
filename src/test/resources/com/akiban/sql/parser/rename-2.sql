RENAME INDEX index1 to index2