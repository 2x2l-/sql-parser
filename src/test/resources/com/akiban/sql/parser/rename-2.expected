com.akiban.sql.parser.RenameNode@2e530cf2
statementType: RENAME INDEX
oldIndexName:
index1
newIndexName: 
index2